disconnected/updating/index.adoc

```diff
@@ -0,0 +1,34 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="about-disconnected-updates"]
+= About cluster updates in a disconnected environment
+include::_attributes/common-attributes.adoc[]
+:context: about-disconnected-updates
+
+toc::[]
+
+A disconnected environment is one in which your cluster nodes cannot access the internet, or where you want to manage update recommendations and release images locally for policy or performance purposes.
+This section covers mirroring {product-title} images, managing an OpenShift Update Service, and performing cluster updates in a disconnected environment.
+
+A single container image registry is sufficient to host mirrored images for several clusters in the disconnected network.
```

Comment:
@skopacz1 @wking Since we removed the description of registry from this section, I think we'd better also remove this line and put it in next section `Mirroring OpenShift Container Platform images`.